templates/post.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html
+  xmlns:th="https://www.thymeleaf.org"
+  th:replace="~{layout :: html(title = |${post.spec.title} + ' - ' + ${site.title}|,  content = ~{::content}, head = null)}"
```

Comment:
```suggestion
  th:replace="~{layout :: html(title = |${post.spec.title} - ${site.title}|,  content = ~{::content}, head = null)}"
```

用了 `||` 就不需要字符串拼接了，否则：
<img width="276" alt="image" src="https://github.com/halo-sigs/halo-theme-obsidian/assets/21301288/ce5ede37-b997-4a17-83a6-127289cb6b36">